src/main/java/me/clip/placeholderapi/commands/impl/local/CommandDump.java

```diff
@@ -167,10 +168,15 @@ private String makeDump(@NotNull final PlaceholderAPIPlugin plugin) {
         .getExpansionsFolder()
         .list((dir, name) -> name.toLowerCase().endsWith(".jar"));
 
-    for (final String jar : jars) {
-      builder.append("  ")
-          .append(jar)
-          .append('\n');
+
+    if (jars == null) {
+      builder.append("  WARN: Jars array was empty!");
```

Comment:
Shouldn't this say something more user friendly like `There were no JARs found in the extensions director`. From what I'm understanding this is what shows under `Expansion Directory:` a big warning that won't make sense for most people seems like the wrong approach.

Reply:
Since this would be printed when the array is null - which would be a rare error - would it perhaps be better to have it changed to `WARN: Could not collect Jar files`

Reply:
Sure. I guess that does make sense. and it is more "user friendly"